Make examples/examples-dev is broken on os x

/kind bug

**What steps did you take and what happened:**
After running `make clean` I ran this:

```
 salazar:cluster-api-provider-aws cha$ make examples-dev
/Applications/Xcode.app/Contents/Developer/usr/bin/make cmd/clusterctl/examples/aws/out/ IMAGE=gcr.io/chuck-heptio/cluster-api-aws-controller:0.0.2
make[1]: *** No rule to make target `cmd/clusterctl/examples/aws/out/'.  Stop.
make: *** [examples-dev] Error 2
```


**What did you expect to happen:**
I expected `make examples` and `make examples-dev` to complete successfully.

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]


**Environment:**

- Cluster-api-provider-aws version: 
- Kubernetes version: (use `kubectl version`): 
- OS (e.g. from `/etc/os-release`):

--- a/Makefile
+++ b/Makefile
@@ -125,19 +125,19 @@ clean: ## Remove all generated files
 	rm -f minikube.kubeconfig
 	rm -f bazel-*
 
-cmd/clusterctl/examples/aws/out/:
+cmd/clusterctl/examples/aws/out:
 	./cmd/clusterctl/examples/aws/generate-yaml.sh
 
-cmd/clusterctl/examples/aws/out/credentials: cmd/clusterctl/examples/aws/out/ ## Generate k8s secret for AWS credentials
+cmd/clusterctl/examples/aws/out/credentials: cmd/clusterctl/examples/aws/out ## Generate k8s secret for AWS credentials
 	clusterawsadm alpha bootstrap generate-aws-default-profile > cmd/clusterctl/examples/aws/out/credentials
 
 .PHONY: examples
 examples: ## Generate example output
-	$(MAKE) cmd/clusterctl/examples/aws/out/ IMAGE=${MANAGER_IMAGE}
+	$(MAKE) cmd/clusterctl/examples/aws/out IMAGE=${MANAGER_IMAGE}
 
 .PHONY: examples-dev
 examples-dev: ## Generate example output with developer image
-	$(MAKE) cmd/clusterctl/examples/aws/out/ IMAGE=${DEV_MANAGER_IMAGE}
+	$(MAKE) cmd/clusterctl/examples/aws/out IMAGE=${DEV_MANAGER_IMAGE}
 
 .PHONY: manifests
 manifests: cmd/clusterctl/examples/aws/out/credentials ## Generate manifests for clusterctl

